types/data.d.ts

```diff
@@ -32,7 +32,9 @@ export const enum Event {
     Performance = 25,
     ScriptError = 26,
     ImageError = 27,
-    LayoutSummary = 28
+    Layout = 28,
+    Crawl = 29,
+    Summary = 30
```

Comment:
>Summary [](start = 4, length = 7)

Summary is too broad. What is it summarizing?

Reply:
It is broad; it's summary for every event that was in the payload. It include the event, start and end. E.g. if were multiple mouse movement events between 323ms and 900ms then the summary event would be [9,232,900].